src/main/java/taxi/controller/driver/AddDriverController.java

```diff
@@ -24,8 +27,18 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp)
     public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOException {
         String name = req.getParameter("name");
         String licenseNumber = req.getParameter("license_number");
+        final String login = req.getParameter("login");
```

Comment:
why `login` should be `final`?
or why other variables are not `final`?

Reply:
it is mvn clean package gives an error message, if login not final